ir/v1.def

```diff
@@ -475,11 +475,13 @@ class AttributeRef : Expression {
 class CompilerOptions;
 #end
 
-class V1Program {
+class V1Program : ISimpleNamespace {
```

Comment:
Yes, that's true. This is only useful for passes that either don't need knowledge of V1-specific IR classes, or are written to support them. The latter especially is what I'm interested in, though there are some existing classes in the former category.